protocol/trojan/trojan.py

```
@@ -25,15 +25,15 @@
 )
 from protocol.vless.vless import check_and_use
 
-RELAY_BUF = 1024 * 1024          # 1 MB — 4× نسبت به قبل
-SOCK_BUF = 4 * 1024 * 1024       # 4 MB بافر سوکت سطح OS
-WRITE_HIGH_WATER = 512 * 1024    # drain فقط وقتی بیشتر از 512KB در بافر باشه
+RELAY_BUF = 4 * 1024 * 1024          # 1 MB — 4× نسبت به قبل
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)